psasim/Makefile

```diff
@@ -0,0 +1,64 @@
+CFLAGS ?=  -Wall -std=c99
+INCLUDE := -I./include/
+DESTDIR ?= /usr/local
```

Comment:
Do we really need to install the library system-wise? Can't we put the shared library to the `psasim` folder instead and link it from there?
Btw I tried `make install` on my Ubuntu workstation and of course it complains:
```
cp libpsaff.so /usr/local/lib/
cp: cannot create regular file '/usr/local/lib/libpsaff.so': Permission denied
```

Reply:
As discussed, this should really just build and run in-place, like we do with the tests